res/css/views/elements/_MessageEditor.scss

```diff
@@ -20,7 +20,7 @@ limitations under the License.
     // this is to try not make the text move but still have some
     // padding around and in the editor.
     // Actual values from fiddling around in inspector
-    margin: -7px -10px -5px -10px;
+    margin: -7px 24px -5px -10px;   // right margin: -10px + 34px
```

Comment:
I wonder if this be done in a less fragile way...

Reply:
(Not a blocking comment as part of the review, I am just wondering...)

Reply:
I applied mx_EventTile_content to the editor, which has the 34px of margin. I had to override the overflow so the autocompletion doesn't get clipped, but I agree this is still better, because less magic in the numbers.